astarte-device-sdk/AstarteDeviceSDK.h

```diff
@@ -22,9 +22,9 @@
 #include <HemeraCore/AsyncInitObject>
 
 #include <HyperspaceProducerConsumer/ProducerAbstractInterface>
+#include <astartetransport.h>
```

Comment:
I have no idea whether this header is exported or not. If it isn't, this will likely break.